### ISSUE TITLE
Can't use "help" subcommand - missing "groff"

### ISSUE THREAD

**Issue by dawid-laszuk-sp** [aws/aws-cli#7964]
### Describe the bug

Can't use `help` subcommand.

### Expected Behavior

Seeing help page when appending `help` to main `aws` or any subcommand, e.g. `aws sagemaker help`.

### Current Behavior

```
❯ aws help

Could not find executable named "groff"
```
```
❯ aws sagemaker help

Could not find executable named "groff"
```

### Reproduction Steps

```
> aws help
```

### Possible Solution

Include `groff` as a dependency for packages.

### CLI version used

aws-cli/2.8.5 Python/3.9.11 Darwin/22.5.0 exe/x86_64 prompt/off

### Environment details (OS name and version, etc.)

MacOs 13.4 (22F66)

**Comment by dawid-laszuk-sp** [comment]
I'd also appreciate not stating "well, actually, just install with `brew install groff`". If this is a required dependency then please update your documentation.

https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html

**Comment by tim-finnigan** [comment]
Hi @dawid-laszuk-sp thanks for reaching out. I created a tracking issue for this a few months ago: https://github.com/aws/aws-cli/issues/7400. As mentioned there, this should now be fixed in v2.8.11 and v1.27.6 via https://github.com/aws/aws-cli/pull/7413. If you run into any issues after [updating](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) your version of v2 please let us know.
